vimrcs/basic.vim

```diff
@@ -152,11 +149,28 @@ if has("gui_running")
     set guioptions-=e
     set t_Co=256
     set guitablabel=%M\ %t
+    " Set bigger font since we usually have big screen now
+    if has('gui_win32')
+      set guifont=DejaVu\ Sans\ Mono:h12
+      " For non-Latin languages, choose available font in 
+      "    your system and set guifontwide below
+      set guifontwide=Microsoft\ Yahei\ Mono:h11
+    else
+      set guifont=DejaVu\ Sans\ Mono\ 12
+    endif
 endif
 
 " Set utf8 as standard encoding and en_US as the standard language
 set encoding=utf8
 
+" Set possible file encodings below, you can add more as you want
+set fileencodings=utf-8,chinese,latin-1,gbk,gb18030,gk2312
+
+" Reload menu *after* set fileencoding can avoid 
+"   garbled characters in non-Latin languages such as Chinese
+source $VIMRUNTIME/delmenu.vim
+source $VIMRUNTIME/menu.vim
```

Comment:
" Reload menu *after* set fileencoding can avoid 
"   garbled characters in non-Latin languages such as Chinese